pyproject.toml

```diff
@@ -7,7 +7,7 @@ allow-direct-references = true
 
 [project]
 name = "multiversx-sdk"
-version = "0.1.0"
+version = "0.9.0"
```

Comment:
Sounds good, since it's latest from here + 1:

https://github.com/multiversx/mx-sdk-py-core/releases